File: JavaScript/exercicio-03/solucao.js
const prompt = require("prompt-sync")();
const readline = require("readline-sync");

function main() {
  
  function olaFuncionario() {

    const nomeFuncionario = prompt("Qual o seu nome ?");
    const mesTrabalhado = prompt("Qual foi o ultimo mês trabalhado ?");
    const valorSalario = parseFloat(prompt("Qual o valor do seu salário ?"));
  
    console.log(`O funcionário ${nomeFuncionario} tem um salario de ${valorSalario.toFixed(2)} reais em ${mesTrabalhado}`);
  
  }
  
  olaFuncionario();

}

main();
readline.question('Pressione Enter para sair ... ');

File: Python/exercicio-03/solucao.py
def main():
    def ola_funcionario():
        nome = input('Qual o nome do funcionario ? ')
        salario = int(input('Qual o valor do seu salario ? '))
        mes_trabalhado = input('Qual foi o ultimo mês trabalhado ?' )
        print(f'O funcionário {nome} tem um salario de {salario}R$ no mês de {mes_trabalhado}')
        


    ola_funcionario()

main()
input('Pressione Enter para sair ...')